### PR TITLE
added changelog for 2.6.1 release

### PR DESCRIPTION
**Pull request by vlad-diachenko**

### CODE REVIEW

**Review by DylanGuedes — APPROVED** [review]
lgtm.

just one thing before relasing this, did we update our docs to tell others how to configure the memberlist labels protection?

**Comment by vlad-diachenko** [comment]
> lgtm.
> 
> just one thing before relasing this, did we update our docs to tell others how to configure the memberlist labels protection?


i do not think that we have a page how to configure memberlist at all. Feel free to add it to main and just backport it to release-2.6.x page and the docs will be automatically published .

**Review by KMiller-Grafana — DISMISSED** [review]

File: docs/sources/release-notes/v2-6.md [hunk 1]
```diff
@@ -22,6 +22,14 @@ As always, please read the [upgrade guide](../../upgrading/#260) before upgradin
 
 ## Bug fixes
 
+### 2.6.1 
+
+v2.6.1 is a patch version for v2.6.0 release that includes following fixes:
+
+- [PR 6658](https://github.com/grafana/loki/pull/6658) Updated version of [dskit](https://github.com/grafana/dskit) and [memberlist](https://github.com/grafana/memberlist) to allow configuring cluster labels for memberlist. Cluster labels prevents mixing the members between two memberlists of separate applications that are run on the same Kubernetes cluster.
+- [PR 6681](https://github.com/grafana/loki/pull/6681) Fixed HTTP connections leak between querier and compactor when Log entry deletion feature is enabled.
+- [PR 6583](https://github.com/grafana/loki/pull/6583) Fixed noisy error messages when Log entry deletion feature is disabled for a tenant
```
[the remark this answers is not in the document]
Comment (09jvilla) [reply docs/sources/release-notes/v2-6.md:31]: Karen, note that the names aren't in the 2.6.0 bug fixes. We can follow whatever format you want going forward, just let us know.

**Review by 09jvilla — APPROVED** [review]
Vlad, I'm approving assuming you accept Karen's changes.

**Pull request merged**